bgcval2/download_from_mass.py

```diff
@@ -655,16 +656,13 @@ def deleteBadLinksAndZeroSize(outputFold, jobID):
                        '12',
                    ])
 
-    elif keys in [
-        [
-            'EXPORT',
-        ],
-        [
-            'export',
-        ],
-    ]:
+    if 'export' in keys:
         medusaMonthlyexport(jobID)
+
     #####
     # Other specific monthly files.
-    else:
-        downloadField(jobID, keys, timeslice='m', dryrun=0)
+#    else:
+#        downloadField(jobID, keys, timeslice='m', dryrun=0)
+
+if __name__ == "__main__":
+    main()
```

Comment:
this is a bit hectic: this is how I'd structure it:
```
if not keys:
...
else:
    if bla in keys:
    ...
    if yada in keys:
    ...
    else:
        finalize
```
Note that you don't need returns since the function is calling other functions as its main purpose

Reply:
This whole keys dict isn't something I've used a lot recently, but I don't want to delete it as the second it goes someone will ask for it back again.

Reply:
I've re-done this, could you have a quick look at the new code. Though note that `downloadField` still isn't working. It's a problem for another day.

Reply:
yeh looking good - I wanted to suggest `elif`'s but I didn't know if those downloading calls are mutually exclusive; the `return` at the top is not quite needed since the function exits if the first `if` is True anyway :grin:

Reply:
Hmmm, good point, they might not be mututally exclusive.

Reply:
Think the most recent changes sort this out now.